node_test.go

```diff
@@ -580,16 +522,12 @@ func TestNodeStart(t *testing.T) {
 
 	{
 		rd := <-n.Ready()
-		if !reflect.DeepEqual(rd, wants[0]) {
-			t.Fatalf("#1: rd = %+v,\n             w   %+v", rd, wants[0])
-		}
+		require.Equal(t, wants[0], rd, "#1")
```

Comment:
The `#1` is unnecessary (here and below), just remove it?

If this assertion fails, the error message will contain the code line. The lines that you marked `#1`, `#2`, etc, are all different, so we'll be able to distinguish which of them failed.

Putting extra info into the message is only necessary in case the line can be called repeatedly.